cli/Valet/Pecl.php

```diff
@@ -43,10 +43,10 @@ class Pecl extends AbstractPecl
      */
     const EXTENSIONS = [
         self::XDEBUG_EXTENSION => [
-            '8.1' => '3.0.4',
-            '8.0' => '3.0.4',
-            '7.4' => '3.0.4',
-            '7.3' => '3.0.4',
+            '8.1' => '3.1.5',
+            '8.0' => '3.1.5',
+            '7.4' => '3.1.5',
+            '7.3' => '3.1.5',
```

Comment:
Thanks @martin-cod 

For PHP 7.3 & 7.4 we should use xdebug v2.9.8 as Magento 2.3 does not support these versions.

https://github.com/weprovide/valet-plus/issues/580#issuecomment-886844867

cc @mischabraam

Reply:
I tested this locally, by using `3.1.5` on PHP 8 and `2.9.8` on PHP 7, it works perfectly

<img width="1351" alt="CleanShot 2022-07-26 at 15 27 51@2x" src="https://user-images.githubusercontent.com/2452991/181017576-abfa123e-2dc7-49fe-9b2b-4d9fd898bc3b.png">

Reply:
For anyone needing it, with the above changes to [cli/Valet/Pecl.php](https://github.com/martinboy/valet-plus/blob/67509c146ffcc5cf9d6b54b9ace5f1dab45b9d3a/cli/Valet/Pecl.php#L45-L50) you can set your xdebug version as needed.
i.e. `~/.composer/vendor/weprovide/valet-plus/cli/Valet/Pecl.php`

For other config changes which are needed for refernce see [z-performance.ini#L64-L78](https://github.com/weprovide/valet-plus/blob/691ae2fce7b463b369d522db59412186efa390dd/cli/stubs/z-performance.ini#L64-L78)

The path to this file on your local system can be found in [phpinfo](https://www.php.net/manual/en/function.phpinfo.php).

Reply:
@rossmc thanks for checking this. On my end I tested xdebug 3.1.5 on Magento 2.4.4 and PHP 7.4. It worked fine. Didn't check Magento 2.3